test/ordered_set_test.dart

```diff
@@ -320,14 +320,24 @@ void main() {
 
       test('sorts after remove', () {
         final orderedSet = OrderedSet<int>();
-        // The initial elements must be in order.
+        // The initial elements must be in order to reproduce the issue.
```

Comment:
what issue? can we elaborate?

Reply:
Tried again, and it wasn't even needed to reproduce the issue, so I just removed the comment.